src/System.Security.Cryptography.OpenSsl/src/System.Security.Cryptography.OpenSsl.builds

```diff
@@ -3,10 +3,10 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <Project Include="System.Security.Cryptography.OpenSsl.csproj">
-      <OSGroup>Windows_NT</OSGroup>
+      <OSGroup>Unix</OSGroup>
     </Project>
     <Project Include="System.Security.Cryptography.OpenSsl.csproj">
-      <OSGroup>Unix</OSGroup>
+      <OSGroup>Windows_NT</OSGroup>
```

Comment:
(By "don't need to split" I mean "I agree with not adding a netstandard1.6 TargetGroup build")